person.rb

```diff
@@ -20,6 +21,10 @@ def correct_name
     @name
   end
 
+  def add_rental(rental)
```

Comment:
- [ ] Rental is a class. In adding a new rental, you have to add the parameters that initialize it.

### Sample code 


 Check this [guide](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship)
 ``` def add_rental(book, date)
    # The person itself should be sent as a parameter to create the new rental achieving the 'has-many' association
    Rental.new(date, book, self)
  end
```